src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentServer.kt

```diff
@@ -125,8 +124,11 @@ interface _LegacyAgentServer {
 
   @JsonRequest("editTask/cancel") fun cancelEditTask(params: TaskIdParam): CompletableFuture<Void?>
 
-  @JsonRequest("editTask/getFoldingRanges")
-  fun getFoldingRanges(params: GetFoldingRangeParams): CompletableFuture<GetFoldingRangeResult>
+  @JsonRequest("editTask/retry")
+  fun retryEditTask(params: EditTask_RetryParams): CompletableFuture<EditTask>
+
+  @JsonRequest("editTask/getTaskDetails")
+  fun getEditTaskDetails(params: TaskIdParam): CompletableFuture<EditTask>
```

Comment:
Move the to the AutoGenerated subset. That way it'll be easy to keep track of which methods have been "migrated" and which ones are pending

Reply:
Done!